CRM/Upgrade/Incremental/php/FiveSixtyFive.php

```diff
@@ -44,6 +44,10 @@ public function upgrade_5_65_alpha1($rev): void {
     $this->addTask('Update New Price Set path', 'updateNewPriceSetPath');
   }
 
+  public function upgrade_5_65_beta1($rev): void {
+    $this->addTask('Drop potentially created unneded priority index on acls', 'dropIndex', 'civicrm_acl', 'index_priority');
```

Comment:
That sentence is... tough. May I suggest either:

* `Drop extraneous SQL index, "civicrm_acl.index_priority"`
* `Drop "civicrm_acl.index_priority"`

Or if you still want to convey some nuance for anyone debugging this task... then leave a comment:

```php
    // This index isn't really needed, and it was only created on some sites. See: dev/core#4472
    $this->addTask('Drop "civicrm_acl.index_priority"', 'dropIndex', 'civicrm_acl', 'index_priority');
```

Reply:
@totten I have updated it now